sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractedReceiptType.cs

```diff
@@ -3,9 +3,16 @@
 
 namespace Azure.AI.FormRecognizer.Models
 {
+    /// <summary>
+    /// </summary>
     public enum ExtractedReceiptType
     {
-        Unrecognized = 1,
-        Itemized = 2,
+        /// <summary>
```

Comment:
A `ReceiptType` also has a confidence.

Reply:
Do we know if `ReceiptType` is a documented enum?

Reply:
It's not in the Swagger.  This came from the original .NET library @ctstone build.  @ctstone , would you be able to share whether or not the receipt type values are documented somewhere?